Sources/RealDeviceMapLib/Modell/Gym.swift

```diff
@@ -271,7 +271,7 @@ public class Gym: JSONConvertibleObject, NSCopying, WebHookEvent, Hashable {
             self.raidSpawnTimestamp = UInt32(fortData.raidInfo.raidSpawnMs / 1000)
             self.raidBattleTimestamp = UInt32(fortData.raidInfo.raidBattleMs / 1000)
             self.raidLevel = UInt8(fortData.raidInfo.raidLevel.rawValue)
-            self.raidIsExclusive = fortData.raidInfo.isExclusive
```

Comment:
funny way to fix this... have you checked your DB before? all rows set to 0?

Reply:
Why is it funny, it’s a bool? No point removing it because stuff may expect it to be there. This is the quickest and simplest way to deal with the fact it’s no longer in the protos - unless you don’t agree?